beacon-chain/blockchain/stategenerator/state_generator.go

```diff
@@ -21,6 +22,13 @@ func GenerateStateFromBlock(ctx context.Context, db *db.BeaconDB, block *pb.Beac
 	if err != nil {
 		return nil, err
 	}
+	log.WithFields(logrus.Fields{
+		"input_block_slot":  block.Slot - params.BeaconConfig().GenesisSlot,
+		"output_state_slot": hState.Slot - params.BeaconConfig().GenesisSlot,
```

Comment:
change these log fields to outputStateSlot (camelCase format)

Reply:
oops I need to delete this, it's mainly for debugging, it serves no purpose in production...